load_map_data.sh

```diff
@@ -21,26 +21,37 @@ map_data_path="${HOME}/data"
 if [ ! -f "${map_data_path}/merged.osm.pbf" ]; then
   mkdir "${map_data_path}"
   cd "${map_data_path}"
-  for filename in massachusetts-latest.osm.pbf rhode-island-latest.osm.pbf new-hampshire-latest.osm.pbf; do
+
+  if [ "${MAP_TYPE}" == "bus" ]; then
     wget --tries=100 --retry-on-http-error=429 --waitretry=100 --random-wait \
-      http://download.geofabrik.de/north-america/us/$filename
-  done
+      http://download.geofabrik.de/north-america/us/massachusetts-latest.osm.pbf
+    mv "${map_data_path}/massachusetts-latest.osm.pbf" "${map_data_path}/merged.osm.pbf"
```

Comment:
At some point we might want to call this `map-region.osm.pbf` instead of `merged.osm.pbf` if we're not always going to be merging multiple datasets into it, but that's not important.